docs/source/usage/wm-widgets.rst

```diff
@@ -44,12 +44,31 @@ Add the following to your :file:`~/.config/qtile/config.py`:
        ),
    ]
 
+.. _bar-usage:
+
+bar
+===
+
+.. note:: There is currently only a script to run inside i3, but you can just strip the
+          i3-specific parts of the binding.
+
+To run the bar simply pipe the output of the binding script into ``bar`` and specify appropriate
+options, for example like this::
+
+    python powerline-bar.py | bar -f "-xos4-*-*-*-*-*-*-*-*-*-*-*-*-*
```

Comment:
Just mentioned this: this code is lacking trailing QUOTATION MARK. Same for the code below I guess.